src/Process/TiledNodeProcessing.js

```diff
@@ -126,7 +126,7 @@ export function processTiledGeometryNode(cullingTest, subdivisionTest) {
 
             if (node.material.visible) {
                 // update uniforms
-                node.setFog(1000000000);
+                node.setFog(context.view.fogDistance);
```

Comment:
I'm not sure it's the best place to store `fogDistance`...
Maybe it would make sense in `Camera` but I have no strong opinion about this.